content/docs/Getting started/deploying-cuebot.md

```diff
@@ -84,6 +84,20 @@ Make sure you also complete the following steps:
     to the database. This configuration is outside of the scope of this
     guide.{{% /alert %}}
 
+1.  Cuebot needs to know the location of the filesystem where render assets
+    are stored and log files are written. In a large-scale deployment a shared
+    filesystem such as NFS is often used for this purpose. For the purposes of
+    this tutorial, you can use any directory you have write access to.
+
+    ```shell
+    export CUE_FS_ROOT="$HOME/opencue-tutorial"
+    mkdir -p "$CUE_FS_ROOT"
+    ```
+
+    If you plan to run [CueGUI](/docs/getting-started/installing-cuegui) and
+    [CueSubmit](/docs/getting-started/installing-cuesubmit) on the same host,
```

Comment:
For the sake of people who are very new to all the terms in the project, it might be helpful to be a bit more explicit and say something like:

If you plan to run CueGUI and CueSubmit on the same host as CueBot, then all three components can use the local filesystem on that host.

How does that sound?

Reply:
Sounds good -- done.